### PR TITLE
GUARD-785 fix dependabot alert

### PR DESCRIPTION
**Pull request by ziammalik**
This is to address dependabot alerts mentioned in this Jira ticket : https://paylease.atlassian.net/browse/RESX-891

### CODE REVIEW

**Review by badboyku — APPROVED** [review]
Huge jump in versions.

File: package.json [hunk 1]
```diff
@@ -36,17 +36,18 @@
...
-    "react-dom": "~15.5.4",
+    "prop-types": "^15.7.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
     "react-hot-loader": "^1.3.0",
-    "webpack": "^1.10.1",
-    "webpack-dev-server": "^1.10.1",
+    "webpack": "^4.44.2",
+    "webpack-cli": "^4.6.0",
+    "webpack-dev-server": "^3.11.0",
```
Comment (techgaun) [inline package.json]: same.. you'll need to fix https://github.com/gozego/react-infinite-scroll-component/blob/master/server.js and https://github.com/gozego/react-infinite-scroll-component/blob/master/webpack.config.js

**Review by techgaun — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ziammalik) [reply package.json]: @techgaun do you want to help me with those? updating these was the only way to get rid of the vulnerability in the package "open", but I lack the knowledge to figure out what else needs to change for these in webpack.config.js and server.js.

**Review by VernSul — APPROVED** [review]

**Review by badboyku — APPROVED** [review]
Way better!!!

File: package.json (hunk 1 above)
Comment (ziammalik) [reply package.json]: I removed the updates to webpack in the latest commit. Should not need the updates to config files now.

**Review by techgaun — APPROVED** [review]

**Pull request merged**